### ISSUE TITLE
yarn test:rest shows an error

### ISSUE THREAD

**Issue by WolfgangT71** [DSpace/DSpace#8204]
**Describe the bug**

yarn test:rest response
yarn run v1.22.17
$ ts-node --project ./tsconfig.ts-node.json scripts/test-rest.ts
Building production app config
Overriding app config with /opt/dspace-angular/config/config.prod.yml
...Testing connection to REST API at https://dspace.umiz.at/server/api...

RESPONSE: 200  

Checking JSON returned for validity...
	"dspaceVersion" = DSpace 7.3-SNAPSHOT
	"dspaceUI" = https://dspace.umiz.at:443
	"dspaceServer" = **https://dspace.umiz.at:443/server**
	"dspaceServer" property matches UI's "rest" config? **false**
	Does "/api" endpoint have HAL links ("_links" section)? true
Done in 4.36s.

in config.prod.yml

**rest:
  ssl: true
  host: dspace.umiz.at
  port: 443**
  nameSpace: **/server**

To my knowledge that should be the same, or what am I missing?

http://dspace.umiz.at:8983/solr/search/select solr is working

**Comment by tdonohue** [comment]
@WolfgangT71 : Again, when asking a question for support, you should be using our Tech Support list (https://groups.google.com/g/dspace-tech/).  I think this is a minor configuration issue again...

My suspicion is that it's simply that, in your backend's `local.cfg`, you don't need to specify port 443 when you are running HTTPS, as that's the default HTTPS port.  So, in your `local.cfg`, just set: `dspace.server.url = https://dspace.umiz.at/server` and see if that fixes things.  It's possible the ":443" is being "dropped/ignored" as that's the default port for HTTPS.  

In other words, the `yarn test:rest` script might be simply expecting "https://dspace.umiz.at/server" but finding "https://dspace.umiz.at:443/server", and that string comparison fails.

**Comment by mwoodiupui** [comment]
If the test is not allowing for explicit specification of default ports, then the comparison is incorrect.

**Comment by WolfgangT71** [comment]
Thanks that was it. At least now all the tests from yarn test:rest pass. Still not be able to login  completely, only get two items, and running points isntead of news page.

**Comment by byteAr** [comment]
![errorDspace7 3](https://github.com/DSpace/DSpace/assets/72761422/e57095cc-97a9-406a-a865-a3233157b0c2)
Hello. Could someone enlighten me with their knowledge? I get this error and I don't know why. Greetings from Argentina

**Comment by byteAr** [comment]
Ejecute el comando yarn test:rest
